.ci/package-testing/Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+#!/bin/groovy
+library 'kibana-pipeline-library'
+kibanaLibrary.load()
+kibanaPipeline(timeoutMinutes: 300) {
+  slackNotifications.onFailure(
+    title: "*<${env.BUILD_URL}|[${params.branch}] Package Testing Failure>*",
```

Comment:
I don't think `params.branch` and `params.commit` are going to exist for this job (they are special in the baseline job). You're probably better off just using all of the defaults here, e.g. just doing `slackNotifications.onFailure {}`. The way the job is likely going to get set up with JJB, it will already have the branch in the name.

Reply:
Pushed with 8df30aa

Reply:
@jbudz you removed it from the wrong Jenkinsfile